HopsanGUI/GUIObjects/GUIModelObject.cpp

```diff
@@ -1055,6 +1056,9 @@ QDomElement ModelObject::saveGuiDataToDomElement(QDomElement &rDomElement)
     QPointF cpos = this->getCenterPos();
     appendPoseTag(xmlGuiStuff, cpos.x(), cpos.y(), rotation(), this->mIsFlipped, 10);
 
+    // Save the alwasys visible setting
+    xmlGuiStuff.setAttribute("visible", mAlwaysVisible);
```

Comment:
Maybe call the attribute "always_visible" to avoid confusion. Since components with this property set to "false" will still be visible in the normal case.

Reply:
I renamed it to "alwaysvisible", without the underscore. Most existing two-word attributes are written without underscore.